### DISCUSSION TITLE
[Bug] vercel部署后，edge浏览器访问，设置Azure OpenAI参数，图片描述极其简单。

### DISCUSSION THREAD

**Discussion by Nilljsp** [lobehub/lobehub#1941] [General Question] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Vercel / Zeabur / Sealos

### 🌐 浏览器

Edge

### 🐛 问题描述

vercel部署后，edge浏览器访问，设置Azure OpenAI参数，图片描述极其简单。
使用Azure原生的测试接口测试，生产内容很丰富。

### 🚦 期望结果

能够返回gpt4v的标准描述，很详细的那种。

### 📷 复现步骤

![cda6ac5044bfdde6e46f0a44b72993f](https://github.com/lobehub/lobe-chat/assets/150798674/24a7b065-3b72-4564-8dbb-97ad9f7c274f)

### 📝 补充信息

![267529d5895e5b9a53b1a54874bbdd3](https://github.com/lobehub/lobe-chat/assets/150798674/d09efe0f-5eb6-4a51-af10-5a2d93a2c905)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
你可能要手动设定下你这个模型的最大输出 token 数，应该是这个问题： https://github.com/lobehub/lobe-chat/discussions/427#discussioncomment-7540361
